crates/astria-composer/src/lib.rs

```diff
@@ -19,7 +19,7 @@
 //! };
 //! # use tracing::info;
 //! # tokio_test::block_on(async {
-//! let cfg = config::get().expect("failed to read configuration");
+//! let cfg = Config::get().expect("failed to read configuration");
```

Comment:
I got a doc test fail on this line.
I fixed it by changing the `config` import on line 17 to `config::Config` and that fixed it.